CHANGELOG.md

```diff
@@ -1,6 +1,7 @@
 # Changelog
 
 ## dev-develop
+ - ENHANCEMENT #20    Added additional receivers for sending notification as CC or BCC
```

Comment:
@rs2487 a feature add something new and a enhancement is a improvement of exist.